docs/DeveloperGuide.md

```diff
@@ -362,18 +445,83 @@ otherwise)
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given command has an invalid syntax.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. PennyWise shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 1.
+
+**Use case: UC4 - Viewing a list**
+
+**MSS**
+
+1. User requests to list entries
+2. PennyWise shows requested list of entries
+3. PennyWise shows graphical overview of entries
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given command has an invalid syntax.
+
+    * 3a1. PennyWise shows an error message.
+
+      Use case resumes at step 1.
+
+**Use case: UC5 - Viewing the summary**
```

Comment:
Summarising statistics instead? viewing might get confused to be part of view command

Reply:
Resolved